lib/ansible/modules/crypto/entrust/ecs_certificate.py

```diff
@@ -768,17 +768,23 @@ def request_cert(self, module):
                     if self.backup:
                         self.backup_file = module.backup_local(self.path)
                     crypto_utils.write_file(module, to_bytes(self.cert_details.get('endEntityCert')))
-                    if self.full_chain_path:
+                    if self.full_chain_path and self.cert_details.get('chainCerts'):
                         if self.backup:
                             self.backup_full_chain_file = module.backup_local(self.full_chain_path)
-                        crypto_utils.write_file(module, to_bytes(self.cert_details.get('chainCerts')), path=self.full_chain_path)
+                        chain_string = ''
```

Comment:
This and the next lines can also be shortened (and IMO simplified) as follows:
```suggestion
                        chain_string = '\n'.join(self.cert_details.get('chainCerts')) + '\n'
                        crypto_utils.write_file(module, to_bytes(chain_string), path=self.full_chain_path)
```